charts/internal/cilium/charts/hubble-relay/templates/deployment.yaml

```
@@ -52,7 +50,15 @@ spec:
           - mountPath: {{ dir .Values.global.hubble.socketPath }}
             name: hubble-sock-dir
             readOnly: true
+          - mountPath: /etc/hubble-relay
+            name: config
+            readOnly: true
+          - mountPath: /var/lib/hubble-relay/tls
+            name: tls
+            readOnly: true
       restartPolicy: Always
+      serviceAccount: hubble-relay
+      serviceAccountName: hubble-relay
```

`serviceAccount` is deprecated, thus you can skip it here.
```suggestion
      serviceAccountName: hubble-relay
```